core/src/test/resources/specmaticConfigFiles/specmatic_exp_dep.json

```diff
@@ -0,0 +1,83 @@
+{
+  "sources": [
+    {
+      "provider": "git",
+      "repository": "https://contracts",
+      "exposes": [
+        "com/petstore/1.spec"
```

Comment:
Can you use yaml instead of spec for the extensions, as .spec is the legacy format.